src/React/SyntheticEvent.purs

```diff
@@ -149,107 +149,118 @@ foreign import data NativeAbstractView :: Type
 foreign import data NativeTouchList :: Type
 
 type SyntheticEvent' r
-  = ( bubbles :: Boolean
-    , cancelable :: Boolean
-    , currentTarget :: NativeEventTarget
-    , defaultPrevented :: Boolean
-    , eventPhase :: Number
-    , isTrusted :: Boolean
-    , nativeEvent :: NativeEvent
-    , target :: NativeEventTarget
-    , timeStamp :: Number
-    , type :: String
-    | r
-    )
+  =
```

Comment:
Move the `=` here onto the same line as `SyntheticEvent'`? Same for all others here.